woocommerce/woocommerce-android
# Adding product types to order creation

Closes: #9783

### Why
We want to see how many orders are created in the app with product extensions after we start supporting them.

### Description
This PR adds the `PRODUCT_TYPES` property to the `ORDER_CREATE_BUTTON_TAPPED` event so we can track all the product types contained in the order created

### Testing instructions
TC1
1. Open the orders, tap
2. Add a new order
3. Add several products
4. Tap on create order
5. Check that the `ORDER_CREATE_BUTTON_TAPPED` event is triggered with the `PRODUCT_TYPES` property

TC2
1. Open the orders, tap
2. Add a new order
3. Add a fee
4. Tap on create order
5. Check that the `ORDER_CREATE_BUTTON_TAPPED` event is triggered WITHOUT the `PRODUCT_TYPES` property

### Images/gif
```
🔵 Tracked: order_create_button_tapped, Properties: {"status":null,"product_types":"variable","product_count":1,"has_customer_details":false,"has_fees":false,"has_shipping_method":false,"blog_id":214253715,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"ecommerce-bundle","is_debug":true,"site_url":"https:\/\/superlativecentaur.wpcomstaging.com"}
```
